pkgs/development/python-modules/qscintilla-qt4/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, disabledIf
+, isPy3k
+, isPyPy
+, pkgs
+, python
+, pyqt4
+}:
+
+disabledIf (isPy3k || isPyPy)
```

Comment:
Btw, there does this limitation come from? I don't see it in the previous code

Reply:
It was just a 100% copy of what was in https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/qscintilla/default.nix -- I didn't do any of the refactoring you're looking into now!

Reply:
Thanks